src/content/language/collections.md

```diff
@@ -277,6 +277,10 @@ and control flow elements.
 *   Control flow element: Conditionally or iteratively adds
     zero or more values to the surrounding collection.
 
+    *   Null-aware element: If a value is not null,
+        inserts the resulting value into the surrounding
+        collection.
```

Comment:
To match the structure of the other leaf elements, maybe: "Evaluates an expression and if the result is not `null`, inserts it into the surrounding collection."

Reply:
Updated!